dockerfiles/startall.sh

```diff
@@ -129,6 +133,9 @@ while [ "$#" -gt 0 ]; do
       m) DEPLOY_PROM=1
          PROMETHEUS_METRICS_SINK_ARG="-f $scriptdir/../helm/examples/metrics/prometheus_metrics.yaml"
          ;;
+      g) DEPLOY_LOKI=1
+         LOG_FORMAT_JSON_ARG="--set global.logging.format="json""
```

Comment:
Not expecting to keep this, but it illustrates how we can declare most advantageous log format for the expected back-end log processor....

Reply:
What happens if this is not specified?

Reply:
no-op, like the other uninitialized vars, however, I'm renaming to LOG_FORMAT_ARG and defaulting to CSV.